optashift-employee-rostering-gwtui/src/main/module.gwt.xml

```diff
@@ -30,4 +30,7 @@
   <!--<set-property name="locale" value="default"/>-->
   <!--<extend-property name="locale" values="en"/>-->
   <!--<set-property-fallback name="locale" value="en"/>-->
+  <set-property name="compiler.stackMode" value="emulated" />
```

Comment:
I'm not familiar with these settings. Do they give readable exceptions in production compiles too?

Reply:
From https://lists.fedorahosted.org/pipermail/rhq-devel/2011-October/001252.html, apparently yes (couldn't find any reference to in the GWT docs, other than http://www.gwtproject.org/javadoc/latest/com/google/gwt/core/server/StackTraceDeobfuscator.html).